[mapdb] deserilisation warning on startup of oh3

While starting oh3 i found in the log the following error for various items. Unfortunately the is no hint which item this is.

```
2020-10-03 18:00:54.178 [WARN ] [ence.mapdb.internal.StateTypeAdapter] - Couldn't deserialize state 'org.openhab.core.library.types.DecimalType@@@100': org.openhab.core.library.types.DecimalType cannot be found by org.openhab.persistence.mapdb_3.0.0.202010010808
```

The error covers different types: DecimalType, DatetimeType, OnOffType

The persistence definition is like this:
```
Strategies {
    everyMinute : "0 * * * * ?"
    everyHour   : "0 0 * * * ?"
    everyDay    : "0 0 0 * * ?"
    default = everyUpdate
}

Items {
    * : strategy = everyChange, restoreOnStartup
}
```

And mapdb is the dfault persitence service.

## Expected Behavior
The warning should simply not occur.

## Your Environment
oh3 build 1952 on a virtual debian 10 machine

Meanwhile i get the following error on a regular base at startup

`2020-10-14 20:03:28.675 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'QueryablePersistenceService.query()' on 'org.openhab.persistence.mapdb.internal.MapDbPersistenceService@2bd77f75': Index 1 out of bounds for length 1
java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
        at org.openhab.persistence.mapdb.internal.StateTypeAdapter.read(StateTypeAdapter.java:48) ~[?:?]
        at org.openhab.persistence.mapdb.internal.StateTypeAdapter.read(StateTypeAdapter.java:1) ~[?:?]
        at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:893) ~[bundleFile:?]
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129) ~[bundleFile:?]
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:888) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:853) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:802) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:774) ~[bundleFile:?]
        at org.openhab.persistence.mapdb.internal.MapDbPersistenceService.deserialize(MapDbPersistenceService.java:169) ~[?:?]
        at org.openhab.persistence.mapdb.internal.MapDbPersistenceService.query(MapDbPersistenceService.java:156) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor41.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [bundleFile:?]
        at org.openhab.core.internal.common.Invocation.call(Invocation.java:52) [bundleFile:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
2020-10-14 20:03:28.718 [ERROR] [ence.internal.PersistenceManagerImpl] - Exception occurred while querying persistence service 'mapdb': Index 1 out of bounds for length 1
java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
        at org.openhab.persistence.mapdb.internal.StateTypeAdapter.read(StateTypeAdapter.java:48) ~[?:?]
        at org.openhab.persistence.mapdb.internal.StateTypeAdapter.read(StateTypeAdapter.java:1) ~[?:?]
        at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:893) ~[bundleFile:?]
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129) ~[bundleFile:?]
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:888) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:853) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:802) ~[bundleFile:?]
        at com.google.gson.Gson.fromJson(Gson.java:774) ~[bundleFile:?]
        at org.openhab.persistence.mapdb.internal.MapDbPersistenceService.deserialize(MapDbPersistenceService.java:169) ~[?:?]
        at org.openhab.persistence.mapdb.internal.MapDbPersistenceService.query(MapDbPersistenceService.java:156) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor41.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [bundleFile:?]
        at org.openhab.core.internal.common.Invocation.call(Invocation.java:52) [bundleFile:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
`

--- a/bundles/org.openhab.persistence.mapdb/pom.xml
+++ b/bundles/org.openhab.persistence.mapdb/pom.xml
@@ -14,6 +14,10 @@
 
   <name>openHAB Add-ons :: Bundles :: Persistence Service :: MapDB</name>
 
+  <properties>
+    <bnd.importpackage>org.openhab.core.library.types</bnd.importpackage>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.mapdb</groupId>

--- a/bundles/org.openhab.persistence.mapdb/src/main/java/org/openhab/persistence/mapdb/internal/MapDbPersistenceService.java
+++ b/bundles/org.openhab.persistence.mapdb/src/main/java/org/openhab/persistence/mapdb/internal/MapDbPersistenceService.java
@@ -102,7 +102,6 @@ public class MapDbPersistenceService implements QueryablePersistenceService {
         if (db != null) {
             db.close();
         }
-        threadPool.shutdown();
     }
 
     @Override

